files/en-us/learn/server-side/first_steps/web_frameworks/index.html

```diff
@@ -178,7 +178,7 @@ <h2 id="How_to_select_a_web_framework">How to select a web framework</h2>
    <li><em>Framework purpose/origin</em>: Some web frameworks were initially created to solve certain types of problems, and remain <em>better</em> at creating web apps with similar constraints. For example, Django was created to support development of a newspaper website, so it's good for blogs and other sites that involve publishing things. By contrast, Flask is a much lighter-weight framework and is great for creating web apps running on embedded devices.</li>
    <li><em>Opinionated vs unopinionated</em>: An opinionated framework is one in which there are recommended "best" ways to solve a particular problem. Opinionated frameworks tend to be more productive when you're trying to solve common problems, because they lead you in the right direction, however they are sometimes less flexible.</li>
    <li><em>Batteries included vs. get it yourself</em>: Some web frameworks include tools/libraries that address every problem their developers can think "by default", while more lightweight frameworks expect web developers to pick and choose solution to problems from separate libraries (Django is an example of the former, while Flask is an example of a very light-weight framework). Frameworks that include everything are often easier to get started with because you already have everything you need, and the chances are that it is well integrated and well documented. However if a smaller framework has everything you (will ever) need then it can run in more constrained environments and will have a smaller and easier subset of things to learn.</li>
-   <li><em>Whether or not the framework encourages good development practices</em>: For example, a framework that encourages a <a href="/en-US/docs/Web/Apps/Fundamentals/Modern_web_app_architecture/MVC_architecture">Model-View-Controller</a> architecture to separate code into logical functions will result in more maintainable code than one that has no expectations on developers. Similarly, framework design can have a large impact on how easy it is to test and re-use code.</li>
+   <li><em>Whether or not the framework encourages good development practices</em>: For example, a framework that encourages a <a href="https://developer.mozilla.org/en-US/docs/Glossary/MVC">Model-View-Controller</a> architecture to separate code into logical functions will result in more maintainable code than one that has no expectations on developers. Similarly, framework design can have a large impact on how easy it is to test and re-use code.</li>
```

Comment:
```suggestion
   <li><em>Whether or not the framework encourages good development practices</em>: For example, a framework that encourages a <a href="/en-US/docs/Glossary/MVC">Model-View-Controller</a> architecture to separate code into logical functions will result in more maintainable code than one that has no expectations on developers. Similarly, framework design can have a large impact on how easy it is to test and re-use code.</li>
```

We don't put the domain into internal links :)